indigo_app/templates/indigo_api/_task_list_all.html

```diff
@@ -0,0 +1,55 @@
+{% load humanize indigo_app %}
+
+{% if tasks %}
+<table class="table table-striped table-bordered">
+  <thead>
+    <tr>
+      <th scope="col">Task</th>
+      <th scope="col">Place</th>
+      <th scope="col">Work/Document</th>
+    </tr>
+  </thead>
+  <tbody>
+  {% for task in tasks %}
+  {% if task.state != 'pending_review' %}
+    <tr>
+      <td>
+        <div class="task-list-span">
+          <i class="fas fa-fw task-icon-{{ task.state }} text-{{ task.state }} small mr-2"></i>
+          <div class="small text-muted mr-2">
+            {% if perms.indigo_api.change_task and task.is_open and selectable%}
+            <label class="mb-0 font-weight-normal c-pointer">
+              <input type="checkbox" name="tasks" value="{{ task.id }}" form="bulk-task-update-form" class="mr-1">
+              #{{ task.id }}
+            </label>
+            {% else %}
+            #{{ task.id }}
+            {% endif %}
+          </div>
+          <div class="text-muted {% if task.labels.all %} mr-2 {% endif %}">{{ task.title }}</div>
```

Comment:
This doesn't need to be muted, and should be a link to the task.

Reply:
Done